### DISCUSSION TITLE
Welcome to gigavibe Discussions!

### DISCUSSION THREAD

**Discussion by ZeNyfh** [ZeNyfh/gigavibe#37] [General]
>Talk about stuff here ig

**Comment by ZeNyfh** [comment]
priority on some stuff was bumped up, expecting to get a stable release within ~3 days.

**Comment by ZeNyfh** [comment]
final bug testing will occur today in approximately 1 hour

**Comment by ZeNyfh** [comment]
development will be slow for the next 2 weeks

**Comment by 9382** [comment]
ill do cleaning while development is slow since i wont have to merge it with anything ur working on

**Comment by ZeNyfh** [comment]
skipped some versions of course due to the amount of changes that had happened from disky 2 to 3.
